server/utils/userUtils.ts

```diff
@@ -33,6 +34,13 @@ export const sections = {
     shortTitle: 'Management information reports',
     href: reportsPaths.report.new.pattern,
   },
+  prisonDashboard: {
+    id: 'prison-dashboard',
+    title: 'View your prison’s applications',
+    description: 'View recently submitted CAS-2 applications from your prison.',
+    shortTitle: 'View your prison’s referrals',
```

Comment:
I notice we've got "referrals" here but "applications" elsewhere, flagging this just in case it wasn't deliberate?

Reply:
ah good spot, it should be applications